geometry_msgs/Point apple_pose
---
bool apple_is_there
